docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -66,7 +66,16 @@ Property Name                              Description
 
 Comma-separated list of ``hostname[:port]`` all MongoDB servers in the same replica set, or a list of MongoDB servers in the same sharded cluster. If a port is not specified, port 27017 will be used.
 
-This property is required; there is no default and at least one seed must be defined.
+This property is required; there is no default and at least one seed must be defined. Seeds or a connection string must be provided to connect to a MongoDB deployment.
+
+``mongodb.connection-url``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A string containing the protocol, credential and host info for use in connection to your MongoDB deployment.
+
+This could be mongodb://user:pass@sample.host:27017/ or mongodb+srv://kay:myRealPassword@cluster0.mongodb.net/
```

Comment:
Let's add a link to the official documentation. 
https://docs.mongodb.com/upcoming/reference/connection-string/